doc/api/worker_threads.md

```diff
@@ -128,10 +128,18 @@ if (isMainThread) {
 added:
   - v14.5.0
   - v12.19.0
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/47604
```

Comment:
nit: isn't this change in postMessage from the API point of view?

Reply:
Yeah, removed.